buildrunner/steprunner/tasks/run.py

```diff
@@ -944,7 +944,8 @@ def run(self, context):  # pylint: disable=too-many-statements,too-many-branches
                 if isinstance(value, str):
                     # get the cache location from the main BuildRunner class
                     cache_local_path = self.step_runner.build_runner.get_cache_path(
-                        key,
+                        cache_name=key,
+                        project_name=os.getcwd().split("/")[-1]
```

Comment:
Is there a better way to get the project name?

Reply:
This provides the following cache name `.buildrunner/caches/buildrunner-tester-venv.tar` from the following syntax in the `buildrunner.yaml`:
```
caches:
        venv: "/root/.m2/repository"
```

Reply:
Probably by using the vcsinfo variables. See where it gets it from in `buildrunner/__init__.py`.